erc-draft_opaque_token.md

```diff
@@ -0,0 +1,210 @@
+---
```

Comment:
```suggestion
---
eip: 7722
```
Assigning next sequential EIP/ERC/RIP number.

Please also update the filename.